includes/admin/class-wc-payments-admin.php

```diff
@@ -70,6 +78,49 @@ public function add_payments_menu() {
 
 		$top_level_link = $should_render_full_menu ? '/payments/overview' : '/payments/connect';
 
+		$this->admin_child_pages = [
```

Comment:
Moving configs to variable is great idea IMO. How about moving it to static const? I don't see it's ever reassigned or modified.

Reply:
I had the same thought but I don't think it's possible. This array calls the `__()` function a few times. If I attempt to make this definition a class constant I receive the following error:

>Fatal error: Constant expression contains invalid operations

As per the [PHP docs](https://www.php.net/manual/ro/language.oop5.static.php):

>Like any other PHP static variable, static properties may only be initialized using a literal or constant before PHP 5.6; expressions are not allowed. In PHP 5.6 and later, the same rules apply as const expressions: some limited expressions are possible, provided they can be evaluated at compile time.

I could potentially move this to the constructor though.

Reply:
Oh, right! I didn't spot this, sorry.

> I could potentially move this to the constructor though.

🤔 I'm not sure how often the class instance is created: with each HTTP call, when any admin page is accessed or only for WC Pay admin page. Maybe setting it to empty array in constructor would be the best alternative to avoid reading from variable before it has value assigned.